README.md

```diff
@@ -2,6 +2,10 @@
 
 This project was bootstrapped with [Nimble React template](https://github.com/nimblehq/react-templates).
 
+# Prerequisites
+
+NodeJS: 16.14.2 LTS
```

Comment:
The current LTS is 18, I think you should update the application to use it as well 🙌